### DISCUSSION TITLE
Adding Debrid client RDT-Client

### DISCUSSION THREAD

**Discussion by velumm** [Luctia/ezarr#59] [Ideas]
Hi, Would it be possible to add in RDT client. I was able to set it up one time on unraid years ago and never really understood the folder path/mapping and its integration to the arrs* would be helpful to integrate in here. There are many people that struggle with rdt-client and arr integration out there

**Comment by Luctia** [comment]
I'm not familiar with RDT client, a quick google search seems to indicate it can be added in Prowlarr, so  I think it should work fine. In its docs it also says "Implements a fake qBittorrent API so you can hook up other applications like Sonarr, Radarr or Couchpotato."

If you want, you can create an issue for this feature and it might get picked up. However, considering I don't have any experience with it, it doesn't have much priority for me. Looking at what I mentioned above, you could look into getting it to work locally (seems to be supported) and make a PR here to add support.
